addons/getLoadoutsFromUnits/functions/fnc_addMedicItems.sqf

```diff
@@ -0,0 +1,106 @@
+params ["_unit", "_type"];
+
+private _log = [];
+private _items = [];
+private _loadout = getUnitLoadout _unit;
+
+{
+   if (!((_loadout select _x) isEqualTo []) && !((_loadout select _x select 1) isEqualTo [])) then {
+      _items append (_loadout select _x select 1);
+   };
+}forEach [3,4,5];
+
+{
+   private _countOfSameItem = 0;
+   private _index = _forEachIndex;
+   {
+      _x params ["_newItem", "_amount"];
+      {
+         if ((_x select 0) == _newItem) then {
+            _countOfSameItem = _countOfSameItem +1;
```

Comment:
We have the `count` command for this.

Reply:
and count is able to count the sub elements of an array with no complicatated additions?